src/modules/Routing.rb

```diff
@@ -137,54 +137,64 @@ def RemoveDefaultGw
       nil
     end
 
+    # Reads current status for both IPv4 and IPv6 forwarding
     def ReadIPForwarding
       if SuSEFirewall.IsEnabled
         @Forward_v4 = SuSEFirewall.GetSupportRoute
-        # FIXME: missing support for setting IPv6 forwarding enablement in
-        # SuSEFirewall module and in SuSEFirewall2 at all
       else
         @Forward_v4 = SCR.Read(path(SYSCTL_IPV4_PATH)) == "1"
-        @Forward_v6 = SCR.Read(path(SYSCTL_IPV6_PATH)) == "1"
       end
 
+      @Forward_v6 = SCR.Read(path(SYSCTL_IPV6_PATH)) == "1"
+
       log.info("Forward_v4=#{@Forward_v4}")
       log.info("Forward_v6=#{@Forward_v6}")
 
       nil
     end
 
-    def WriteIPForwarding
-      forward_ipv4 = @Forward_v4 ? "1" : "0"
-      forward_ipv6 = @Forward_v6 ? "1" : "0"
+    # Configures system for IPv4 forwarding
+    #
+    # @param [Boolean] true when forwarding should be enabled
+    def write_ipv4_forwarding(forward_ipv4)
+      sysctl_val = forward_ipv4 ? "1" : "0"
 
       if SuSEFirewall.IsEnabled
-        # FIXME: missing support for setting IPv6 forwarding enablement in
-        # SuSEFirewall module and in SuSEFirewall2 at all
-        SuSEFirewall.SetSupportRoute(@Forward_v4)
+        SuSEFirewall.SetSupportRoute(forward_ipv4)
       else
         SCR.Write(
           path(SYSCTL_IPV4_PATH),
-          forward_ipv4
-        )
-        SCR.Write(
-          path(SYSCTL_IPV6_PATH),
-          forward_ipv6
+          sysctl_val
         )
         SCR.Write(path(".etc.sysctl_conf"), nil)
       end
 
-      SCR.Execute(
-        path(".target.bash"),
-        "echo #{forward_ipv4} > /proc/sys/net/ipv4/ip_forward"
-      )
-      SCR.Execute(
-        path(".target.bash"),
-        "echo #{forward_ipv6} > /proc/sys/net/ipv6/conf/all/forwarding"
```

Comment:
yes ```/proc``` was for immediate action. But I've replace that using ```sysctl -w``` call. I like it more bcs it hides dependency on /proc and is more consistent with updating persistent sysctl config few lines above.